mo_net/samples/cbow.py

```
@@ -0,0 +1,426 @@
+    return wrapper
+
+
+@click.group()
+def cli():
+    """CBOW (Continuous Bag of Words) model CLI"""
+    pass
+
+
+@cli.command("train", help="Train a CBOW model")
+@training_options
+def train(
```

Pylint (3.3.7)

[refactor] Too many arguments (9/5)

(R0913)